docker-compose.yml

```diff
@@ -7,3 +7,19 @@ services:
       dockerfile: sfeth.Dockerfile
     ports:
       - 13042:13042
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:8547"]
```

Comment:
sfeth opens this port on startup 🚢

Reply:
I don't get what this command does.

The port mapping i added `- 13042:13042` enables me to call the grpcurl command locally outside the container. Does this have a different purpose?

Reply:
It exists to delay the startup of the grpcurl service. if both services start at once grpcurl blows up because sfeth hasnt had time to properly start. The reason i used this port instead of 13042 is because it looks like sfeth doesnt set any REST implementation on that port, returns a 503 when i curl it. I used 8547 because i saw in the logs that sfeth sets it up on startup, and it would be easier to ping this instead of creating my own healthcheck endpoint

Reply:
Ah this makes sense. The node healthcheck is running on a different port than firehose